pyscriptjs/src/runtime.ts

```diff
@@ -0,0 +1,177 @@
+import type { loadPyodide } from 'pyodide';
+import type { PyLoader } from './components/pyloader';
+import {
+    runtimeLoaded,
+    loadedEnvironments,
+    globalLoader,
+    initializers,
+    postInitializers,
+    Initializer,
+    scriptsQueue,
+    appConfig
+} from './stores'
+import type { PyScript } from './components/pyscript';
+
+// The current release doesn't export `PyodideInterface` type
```

Comment:
I'd add that it's referring to the current **pyodide** release and not PyScript release

Reply:
fixed it, the newer version exports it